guides/common/attributes-base.adoc

```diff
@@ -18,12 +18,24 @@
 // The rest of repo IDs moved to attributes-satellite.adoc
 :SatelliteSub: Red Hat Satellite Infrastructure Subscription
 :RepoRHEL7Server: rhel-7-server-satellite
-:RepoRHEL8BaseOS: rhel-8-for-x86_64-baseos-rpms 
+:RepoRHEL8BaseOS: rhel-8-for-x86_64-baseos-rpms
 :RepoRHEL8AppStream: rhel-8-for-x86_64-appstream-rpms
 :RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
-:RepoRHEL8ServerAnsible: ansible-2.9-for-rhel-8-x86_64-rpms 
+// For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.11-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.9-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.11-rpms
+:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.10-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6.11-rpms
+// For RHEL 8
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-6.11-for-rhel-8-x86_64-rpms
+:RepoRHEL8ServerSatelliteMaintenanceProductVersion: satellite-maintenance-6.11-for-rhel-8-x86_64-rpms
+:RepoRHEL8ServerSatelliteCapsuleProductVersion: satellite-capsule-6.11-for-rhel-8-x86_64-rpms
+// Do not update the puppet4 repo versions. They must stay at 6.3.
+:RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
+:RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
```

Comment:
I think all these should go into `guides/common/attributes-satellite.adoc`, not into `guides/common/attributes-base.adoc`

Reply:
actually, they already are?

https://github.com/theforeman/foreman-documentation/blob/e18c3c19b7a5472daa7c3c69a8bea1c70e5d32eb/guides/common/attributes-satellite.adoc#L101-L110

Reply:
Yes. Those attributes are covered in `attributes-satellite`. Removed from `attribute-base'. Thank you for this.